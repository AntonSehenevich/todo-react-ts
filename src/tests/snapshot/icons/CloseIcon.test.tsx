import renderer from 'react-test-renderer'
import CloseIcon from '../../../components/icons/close'

test('renders correctly', () => {
  const tree = renderer
    .create(<CloseIcon width={10} height={10} handleClick={jest.fn()} />)
    .toJSON()

  expect(tree).toMatchSnapshot()
})
