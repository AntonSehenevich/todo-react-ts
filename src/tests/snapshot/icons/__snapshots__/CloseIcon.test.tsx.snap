// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<svg
  className="icon"
  data-testid="close-icon"
  height={10}
  onClick={[MockFunction]}
  viewBox="0 0 50 50"
  width={10}
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M 5 5 L 45 45 M 45 5 L 5 45"
    fill="transparent"
    stroke="currentColor"
    strokeLinecap="round"
    strokeWidth="3"
  />
</svg>
`;
